Makefile

```diff
@@ -18,11 +18,11 @@ run-build:
 	@${ps-bundle} && BROWSER_RUNTIME=1 webpack --mode=production
 
 check-explicit-exports: $(eval SHELL:=/bin/bash)
```

Comment:
Is this actually necessary, given that we have `SHELL := bash` earlier?

Reply:
No